index.js

```diff
@@ -92,6 +92,9 @@ var tslintPlugin = function (pluginOptions) {
                 return cb(error, undefined);
             }
             var linter = getTslint(pluginOptions);
+            if (pluginOptions.configuration == null || typeof pluginOptions.configuration === "string") {
+                options.configuration = linter.findConfiguration(pluginOptions.configuration);
```

Comment:
We should actually remove all references to Rcloader and use ``.findConfiguration`` instead of that. In addition, we should pass the location of the file being linted as the second argument to ``.findConfiguration`` (see https://github.com/palantir/tslint/blob/master/src/configuration.ts#L71 ).